substrate/frame/broker/src/utility_impls.rs

```diff
@@ -94,11 +94,12 @@ impl<T: Config> Pallet<T> {
 	pub fn issue(
```

Comment:
The more I think about this the more I think this should just be `pub(crate)` - I see no reason for this to be exposed as `pub`
```suggestion
	pub(crate) fn issue(
```

Reply:
no wait, ignore this